src/util/validators/testCode.ts

```diff
@@ -0,0 +1,16 @@
+const testCodeMaps = [
+  {
+    original: 'NVT',
+    replacement: 'NPT',
+  },
+  {
+    original: 'NVV',
+    replacement: 'NPV',
+  },
+];
+
+export const validateTestCode = (testCode: string): string => {
+  return (
+    testCodeMaps.find((x) => x.original === testCode)?.replacement ?? testCode
+  );
+};
```

Comment:
Something simpler could be
```
const testCodeMaps: {[key: string]: string | undefined} = {
    'NVT': 'NPT',
    'NVV': 'NPV'
}

const validateTestCode = (testCode: string): string => testCodeMaps[testCode] ?? testCode
```

Reply:
In this case I wouldn't even wrap `testCodeMaps[testCode] ?? testCode` in a function

Reply:
I can't decide which way I prefer. Keep changing my mind!

Reply:
Did it.